phac-nml/irida
# Fix user group removal from project

## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Fixed removal of user groups that have been added to projects. Before this fix the user group could not be removed if it was linked to any projects

**To test these changes:**

First create a few groups and a few projects. Then add groups to these projects. Go to the groups page and click that trash can icon next to any of the groups that you have added to a project to delete this group. You should get a popup with a warning about the group being linked to a project or projects. Clicking the "Remove Group" button should delete the group.

Note: Try this out with a group linked to no projects and linked to 1,3, and 4 projects. You should receive the correct translations depending on the number of projects linked to the group.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

Fixes #115

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~~* [ ] User documentation updated for UI or technical changes.~~
